osgp/platform/osgp-throttling-service/src/main/java/org/opensmartgridplatform/throttling/PermitsPerNetworkSegment.java

```diff
@@ -100,22 +103,57 @@ public Map<Integer, Map<Integer, Integer>> permitsPerNetworkSegment() {
                 TreeMap::new));
   }
 
+  public ConcurrentMap<Integer, ConcurrentMap<Integer, NewConnectionRequestThrottler>>
+      newConnectionRequestThrottlerPerSegment() {
```

Comment:
Ik zie geen gebruik van deze methode.
En de Map wordt leeg geinitialiseerd en verder ook niet gevuld.

Reply:
Deze gebruik ik in de test, heb de methode package-private gemaakt

Reply:
Ok. Ben zelf van het testen via de publieke API, maar je zal je redenen wel hebben.